sdks/python/apache_beam/ml/inference/test_triton_model_handler.py

```diff
@@ -0,0 +1,409 @@
+import unittest
+from unittest.mock import patch, MagicMock, ANY, call
+import json
+from google.cloud import aiplatform
+from apache_beam.ml.inference.vertex_ai_inference import VertexAITritonModelHandler
+from apache_beam.ml.inference import utils 
+from apache_beam.ml.inference.base import PredictionResult
+import numpy as np
+import base64 
```

Comment:
import order is wrong, the linting/formatting checks should have the correct order listed but for reference you should be importing in at least two distinct blocks: native python imports first, then third-party imports. These should be in alphabetical order within each block as well.